wikimedia/wikipedia-ios
# Update content license to 4.0

**Phabricator:**
https://phabricator.wikimedia.org/T341144

### Notes
This updates our CC-BY-SA content license callouts to 4.0. I noticed Android's PR used a new string key for new translations, so I'm doing the same here.

### Test Steps

Confirm license shows and links to 4.0 in:

1. Settings > About the app
2. Talk pages > new topic and talk pages > reply
3. Editor flow > edit summary view
